themes/default/content/blog/pulumi-service-updates/index.md

```diff
@@ -0,0 +1,104 @@
+
+We knew we could do better. Now, we're excited to share the changes we've made so far!
+
+## New primary navigation pane
+
+Next time you sign in to the Pulumi Service, look for the navigation pane on the left hand side of the page. All primary navigation items like your Dashboard, Stacks, and Settings are accessible in the new pane from anywhere in the Service.
+
+![""](dashboard-nav-expanded.png)
+
+The navigation pane consolidates actions and information so you don't have to look so hard to find where things are. You'll find options in the pane to:
+
+- switch between your individual account and any other organizations you're part of
+- create new organizations
+- access your favorite stacks (read on to find out more!)
+- directly access sub-pages like Billing or Access Tokens
+- send us your feedback!
+
+![""](dashboard-nav-collapsed.png)
+
+We made these changes to help users to move through the Pulumi Service without losing their place and ensure constant access to key pages. Even if you collapse the navigation pane for more screen real estate, you can still get where you need to go!
+
+## Stack Favorites
+
+The Pulumi Service now supports stack favorites. We want to reduce the time practitioners spend combing through projects. Stack favorites offer quick access to the stacks you need most. Look for the star icon next to any stack name and select it to start building out your favorites list.
+
+![""](stack-navigation.png)
+
+Your favorites list is your own—even if you're part of an organization, your favorites won't be shared with anyone other than you. And you can have as many favorites as you like!
+
+You can access the favorites list from two key locations in the Service: the primary navigation pane and the dashboard. All your favorites will appear in the navigation pane under the “Stacks” heading. You'll also have your list of favorites in a new dashboard card.
+
+![""](stacks-navigation-favorites-card.png)
+
+## Pulumi Service dashboard redesign
+
+linter fun _image will be added here_
+
+Lastly, we’ve also made significant changes to the Pulumi Service dashboard. Now you can access more information to help survey your operations at a glance.
+
+Organizations can access some quick usage statistics at the top of their dashboards:
+
+- member count and +/- change in the past week
+- total stack count and percent change in the past week
+- total resource count and percent change in the past week
+- number of successful updates in the past day, week, month, and year
+
+![""](dashboard-quick-stats.png)
+
+Users in organizations will also see a “Members” card that tracks recently joined members. Admins can manage any pending invites from this card.
+
+All users can now see more of their operation updates in the “Latest Stack Activity” card. You can expose up to ten updates and favorite stacks directly from the dashboard.
```

Comment:
there is inconsistent use of first person and third person (users) throughout this blog post. recommend choosing one and making it consistent throughout